<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newsketch_screen</class>
 <widget class="QDialog" name="newsketch_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Sketch Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>9</y>
     <width>79</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
  </widget>
  <widget class="QListWidget" name="controllerTypeList">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>32</y>
     <width>256</width>
     <height>311</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>T-1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-3000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-400K</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-500K</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-600K-D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-700K</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-700K-B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-780</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-790</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T-800K</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-1000-RGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-1000-RGBW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-1000-W</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-1000-RGBWYA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-8000-RGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-8000-RGBW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-8000-W</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-8000-RGBWYA</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>113</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Controller Type :</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>32</y>
     <width>454</width>
     <height>306</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="currentControllerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Controller :</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>299</width>
      <height>39</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max number of light</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lightNumberlineEdit">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>(1-2048)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Path of the sketch file :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="broweButtonSketchPath">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Path of byte file : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButtonBytePath">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sketchPathInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bytePathInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openSketchpushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Another Project</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newsketch_screen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newsketch_screen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
